.config/jest.config.js

```diff
@@ -12,8 +12,8 @@ const readFile = promisify(fs.readFile)
 const DefaultRunCfg = {
     projectPath: `${HOME_DIR}`,
     match: [
-        `${HOME_DIR}/tests/?(*.)+(spec|test).[jt]s?(x)`,
-        `${HOME_DIR}/tests/**/?(*.)+(spec|test).[jt]s?(x)`
+        `${HOME_DIR}/tests/?(*.)+(spec|test).js?(x)`,
```

Comment:
I guess this is something we should adjust for all images then?

Reply:
@alexplischke not the TestCafe image. TestCafe accepts .ts as-is

Reply:
@dpgraham do we not expect any `.ts` files?

Reply:
@christian-bromann you can't run .ts files directly, they need to be transpiled first.